app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const app = express();
+app.use(express.json());
```

Comment:
We're not doing a post request yet so this line isn't needed yet and should be removed because it's good practise to only include code in a pull request that achieves that ticket 🙂